ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -28,7 +30,7 @@ public void checkModulesTest() {
         for (WebElement eachModule : firstColumn) {
             // The name attr on the <input> elements should all be "stop" which indicates the module is correctly started.
             // If not, then grab the text from the 3rd column to show which module is not started.
-            Assert.assertEquals("module not ready: " + eachModule.findElement(By.xpath("../../td[3]")).getText(), "stop", eachModule.getAttribute("name"));
+            assertFalse(eachModule.getText().contains("moduleNotStarted"));
```

Comment:
@kdaud in the AssertEquals the `actual: eachModule.getAttribute("name")` was returning null 

and due to `// If not, then grab the text from the 3rd column to show which module is not started.` data why changed it

Reply:
@kdaud `assertEquals(java.lang.String message, java.lang.Object expected, java.lang.Object actual)` so [this comment](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/581#issuecomment-934165703) even if i change it it is just the message return as u see it look and returns the string of the module failing or not started..................and as said [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/581#discussion_r725997885) that `eachModule.getAttribute("name") ` which is the actual is already returning null

Reply:
@kdaud This may be about the version of ReferenceApplication running cause when you look for `Attribute name = stop` ........is found in local where u can load and unload the module bt in qa-server u cant unload since they a locked